cmd/fluxd/main.go

```diff
@@ -224,17 +239,61 @@ func main() {
 		os.Exit(0)
 	}
 
+	// Configure viper to check for a environment variables or a config file;
```

Comment:
do we really want to enable env variables? I guess it's one env variable per flag, which seems overkill.

Also, the configuration file may end up evolving into something which isn't flag (e.g. when supporting multiple git repositories) point at which it wouldn't make a lot of sense to use env variables anymore.

Reply:
The original PR included that and I assumed it was part of the desired change; I don't myself see any strong argument for having everything set by environment variable. It would simplify things to remove it (and it can always be added later, now the original PR has shown us how)

Reply:
The original PR included that and I assumed it was part of the desired change; I don't myself see any strong argument for having everything set by environment variable. It would simplify things to remove it (and it can always be added later, now the original PR has shown us how)